### ISSUE TITLE
Can't mount non-C/system drive from Windows (to Windows)

### ISSUE THREAD

**Issue by csm10495** [billziss-gh/sshfs-win#225]
I have Microsoft's OpenSSH server running on a Windows 10 PC. On my laptop I'm trying to use sshfs-win to map the Z: drive on the server PC to the Z on my laptop.

```
net use Z: \\sshfs.r\user@host!port\Z:
```

leads to a System error 67.

If i remove \Z: (so mount root directly) it can mount but i think the colon in the name of the external drive messes it up:
![image](https://user-images.githubusercontent.com/5749838/90327368-e112d000-df47-11ea-944b-1531320993ef.png)

If I go the drive after mounting via the root, powershell's dir looks confused
![image](https://user-images.githubusercontent.com/5749838/90327410-2a631f80-df48-11ea-83d8-2befa6858a4d.png)

Any ideas? I think the colon after the drive letter is upsetting it.

**Comment by billziss-gh** [comment]
Yes. The colon in not a legal character in file names in Windows. So it is mapped to a character in the [Unicode PUA](https://en.wikipedia.org/wiki/Private_Use_Areas).

**Comment by csm10495** [comment]
@billziss-gh , so is there a way to mount the Z drive in my case?

**Comment by billziss-gh** [comment]
You may need to use the sshfs-win.exe command line to mount. Unfortunately you cannot use the `net use` method on a path with a colon currently.

**Comment by csm10495** [comment]
:/. Fair enough. Could there be some sort of escape sequence for colons? Or an alternative addressing like /Z to get to Z: ?

**Comment by billziss-gh** [comment]
I think that this would be the right solution.

I favor using `$` as in `Z$`, but I note that we are already using `!` instead of `:` to specify a port, so perhaps we should also use `!` as in `Z!`.

**Comment by csm10495** [comment]
I think staying consistent with ! Is better though either way is fine.

**Comment by ardent42c** [comment]
Anything implemented to fix this? 
Couldn't find any way to remove the Colon in window's OpenSSH SFTP config either..

**Comment by ardent42c** [comment]
A working workaround that I've found:
On the remote server, under the user profile, create directory links to any drive like this: 
     cd c:\users\user1
     mklink /D /J DRIVED d:\
and then you can map that folder under the user profile as a drive:
     net use x: \\sshfs\user1@myserver\drived password
